README.md

```diff
@@ -1,2 +1,63 @@
 # Terraform Provider Xray
-[Xray API Documentation](https://www.jfrog.com/confluence/display/JFROG/Xray+REST+API)
\ No newline at end of file
+
+[![Actions Status](https://github.com/jfrog/terraform-provider-xray/workflows/release/badge.svg)](https://github.com/jfrog/terraform-provider-xray/actions)
+[![Go Report Card](https://goreportcard.com/badge/github.com/jfrog/terraform-provider-xray)](https://goreportcard.com/report/github.com/jfrog/terraform-provider-xray)
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/jfrog/terraform)
```

Comment:
Since none of us are actively on this Gitter, I think we should remove it from the README

Reply:
ok, removed